import-service/src/handlers/importProductsFile.ts

```diff
@@ -0,0 +1,28 @@
+import { buildResponse } from '../utils';
+import { S3 } from 'aws-sdk';
+
+export const handler = async (event: any) => {
+    const s3 = new S3();
+    const bucketName = process.env.BUCKET_NAME!;
+    const filePath = event.queryStringParameters.name;
+
+    if (!bucketName) {
+        return buildResponse(500,
```

Comment:
This shouldn't be in response rather, whereas fileName validation   would be appreciated

Reply:
Serhiy, I've left comment under Cross-Check: Submit review